docs/troubleshooting.md

```diff
@@ -19,3 +19,18 @@ You recently downloaded or updated XCode.
 2. Choose `Legacy Build System`
 3. Remove `node_modules/react-native/third-party` and
    `node_modules/react-native/third-party-podspecs`
+
+## I get Android crashes when using the Reader SDK
+
+### The problem
+
+When there are different versions of Google Play Services, crashes can occur like
+`java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/android/gms/common/api/Api$zzf;`
+
+### Likely cause
+
+You are using other libraries that use `googlePlayServicesVersion` at something other than "12.0.1"
```

Comment:
This plugin take "googlePlayServiceVersion" (no 's' after 'Service'), do you have other dependencies are actually overridden by 'googlePlayService*s*Version' ? How about you try to override the 'googlePlayServiceVersion'(no 's') to a newer version and see if that works?

I'm asking to do this because I don't think we should introduce another build variable that this plug doesn't depend on. 

Alternatively, can you just list your dependencies that conflict with reader sdk, and provide a solution based on that?

Reply:
+1 to providing specific details about the environment in which the conflict occurs so it's easier for folks to tell if the possible solution applies to their environment as well.